website/src/app/kb/user-guides/ios-client/readme.mdx

```diff
@@ -40,7 +40,9 @@ use it to securely connect to Resources.
 To copy-paste the address of a Resource:
 
 1. Tap on the Resource.
+1. Long-press `ADDRESS`.
 1. Tap `Copy Address`.
+1. Open your web browser.
 1. Long-press your browser's URL bar and tap `Paste and Go`
```

Comment:
I noticed these changed due to the recent resources menu changes

Reply:
Good catch.